libvast/src/system/spawn.cpp

```diff
@@ -79,10 +82,28 @@ expected<actor> spawn_exporter(local_actor* self, options& opts) {
     anon_send(exp, extract_atom::value, limit);
   else
     anon_send(exp, extract_atom::value);
+  if (has_continuous_option(query_opts)) {
+    self->request(self->state.tracker, infinite, get_atom::value).then(
+      [=](registry& reg) mutable {
+        VAST_DEBUG("Looking for importers");
+        // TODO: should probably not restricted to the local node?
```

Comment:
Just looking at the local node is fine here, for now. I picture that each NODE will have its own continuous query in a distributed setting. We'll see.